pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss

```diff
@@ -237,6 +237,12 @@ hr {
   position: relative;
   overflow: hidden;
 
+  & > .grid-container {
```

Comment:
I added this here so we can use this on hero banners going forward. This will reduce the amount of custom css needed for these.